### PR TITLE
Visitor can filter articles by category

### PR DESCRIPTION
**Pull request by kermit-klein**
[PT LINK](https://www.pivotaltracker.com/story/show/172988077)

- Add new fixture for category test   
- Add test file for visitor can see category feature  
- Add navbar with categories ( current other sport politics economy world entertainment)
- Add date comparison for current news category ( it includes last 24 hours news )  
- Add routes for categories ( /category/:category )
Screenshot from 2020-05-24 13-39-52

Note: We are aware of possible merge conflict after Visitor_can_see_ads PR, we plan to solve it before this PR merge
Note : Marcus and I worked on my Vscode live share

### CODE REVIEW

File: src/components/Header.jsx [hunk 1]
```diff
@@ -1,22 +1,17 @@
 import React from "react";
-import { Grid } from "semantic-ui-react";
+import { Grid, Menu, Icon } from "semantic-ui-react";
```
Comment (ViaMarcus) [inline src/components/Header.jsx]: These are not used

File: src/App.jsx [hunk 2]
```diff
@@ -1,22 +1,21 @@
 import React from "react";
-import { 
-  BrowserRouter as Router,
-  Switch,
-  Route
-} from 'react-router-dom';
+import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
```
Comment (ViaMarcus) [inline src/App.jsx]: Router still not used in this file

**Review by designerofthing — APPROVED** [review]
Nice styling for the navbar

File: src/components/ArticleList.jsx [hunk 3]
```diff
@@ -19,14 +19,21 @@ const ArticleList = () => {
...
-    return (
-      <ArticleCard
-        article={article}
+  let filteredArticles = () => {
+    switch (category) {
+      case "":
+        return articleList;
+      case "current":
+        return articleList.filter((article) => {
+          return Date.now() - Date.parse(article.published_at) < 86400000;
```
Comment (erikbjoern) [inline src/components/ArticleList.jsx:28]: 86400000 milliseconds is equal to 24h, for anyone reading this wondering. I just googled it.

**Review by erikbjoern — APPROVED** [review]
It will probably need to resolve merge conflicts after the other PR is merged, but this looks good!

**Review by oliverochman — APPROVED** [review]

**Pull request merged**